docs/source/guide/prompts_predictions.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Generate predictions from a prompt
+short: Generate predictions from a prompt
+tier: enterprise
+type: guide
+order: 0
+order_enterprise: 234
+meta_title: Generate predictions
+meta_description: Generate predictions to auto-label project tasks
+section: Prompts
+date: 2024-06-13 21:54:22
+---
+
+You can continue to save and evaluate your prompts until you reach your desired level of accuracy. When your prompt is ready, you can use it to generate predictions for all tasks within your project. 
+
+!!! note
```

Comment:
i'm not sure i understand this note? would i be able to get some more context?

Reply:
I don't think I wrote the note well, and I was missing a bit of context. I was trying to explain why their ground truth tasks might have multiple predictions. 
For example, in the demo project I'm currently working in, each ground truth task I have has 3 predictions:

1 prediction is from Model A, from when I clicked "Evaluate"
1 prediction is from Model B, from when I clicked "Evaluate"
1 prediction is from Model A, from when I clicked "Get Predictions for All Tasks" <-- this scenario I didn't include, because I'm not sure if that's the expected behavior.

Reply:
So yeah, this is the expected behavior at the moment (and I think it's worth including a note about it). In the future we want to improve this behavior, but doing it differently gives different weird user experiences, so this is kind of the lesser of the evils atm.
I think phrasing this by directly pointing out the weirdness might be helpful, something like:

"Evaluating your Prompts can result in multiple predictions on your tasks: if you have multiple Prompts for one Project, or if you click both "Evaluate" and "Get Predictions for All Tasks" from a Prompt, you will see multiple predictions for tasks in the Data Manager"

Wdyt?

Reply:
Might even be worth making this a "warning" admonition.